src/winsign/sign.py

```diff
@@ -52,6 +52,7 @@ async def sign_file(
     digest_algo,
     certs,
     signer,
+    cafile,
```

Comment:
We should probably make this optional, and throw an error if we're not on MSIX mode. This doesn't end up getting used if `is_msix` is False.